### PR TITLE
Refactor rooms page layout

### LINKED ISSUE #641: Refactor rooms page layout
With the new features coming up (heatmap, current / up-next groups) the room cards on the front page are getting taller. With much of the area being empty on the middle I think we can change the layout to a two-column one like at [Mosogep.sch](https://mosogep.sch.bme.hu).

### PR DESCRIPTION
**Pull request by OmTheTurtle**
Closes #641

### CODE REVIEW

**Comment by OmTheTurtle** [comment]
Available to test at https://tanulo.kir-dev.sch.bme.hu/

**Review by balintkiraly — APPROVED** [review]

**Review by Tschonti — APPROVED** [review]

File: views/room/index.pug [hunk 1]
```diff
@@ -4,16 +4,18 @@ include roomTemplate.pug
...
+      if user && user.floor && user.floor >= 3 && user.floor <= 18
+        div(class='pt-3')
+          if busyRooms.some(it => it.room === user.floor)
+            - const group = busyRooms.find(it => it.room === user.floor)
+            +roomTemplate(user.floor, group, usageData.get(user.floor))
+          else
+            +roomTemplate(user.floor, false, usageData.get(user.floor))
+    if user && user.floor
+      div(class='h-px max-w-3xl mx-auto mt-8 mb-6 bg-gray-700 rounded')
+    div(class='grid grid-cols-1 lg:grid-cols-2 gap-y-8 gap-x-8')
```
Comment (triszt4n) [inline views/room/index.pug:18]: I've been experimenting a little bit. Maybe we could use `md:grid-cols-2` in the first inner divs, they don't feel too bad on md breakpoint too.

Grid-cols-2 starting from lg (feels too spacious and wide):
oie_kjT9tMXxlXhP

Grid-cols-2 starting from md:
oie_Ik4ybDqP021g

Sorry for bad cropping :(

**Review by triszt4n — COMMENTED** [review]
Noice, but I've tried something, see below

File: views/room/index.pug (hunk 1 above)
Comment (OmTheTurtle) [reply views/room/index.pug:18]: I like the more spacious version. Especially if any current / upcoming event is displayed.

Comment (triszt4n) [reply views/room/index.pug:18]: > I like the more spacious version. Especially if any current / upcoming event is displayed.

True, forgot about them. 👌

**Review by triszt4n — APPROVED** [review]

**Pull request merged**